.taskcluster.yml

```diff
@@ -58,11 +58,38 @@ tasks:
           owner: cdenizet@mozilla.com
           source: ${repository}/raw/${head_rev}/.taskcluster.yml
 
+      - taskId: {$eval: as_slugid("windows_test_task")}
+        created: {$fromNow: ''}
+        deadline: {$fromNow: '1 hour'}
+        provisionerId: proj-relman
+        workerType: win2012r2
+        payload:
+          maxRunTime: 3600
+          command:
+            - set RUST_INIT_PATH=%CD%\rust-init
+            - set PATH=%RUST_INIT_PATH%;%PATH%
+            - rustup-init -yv --default-toolchain stable --default-host x86_64-pc-windows-msvc
+            - git clone --recursive --quiet ${repository}
+            - cd rust-code-analysis
+            - git -c advice.detachedHead=false checkout ${head_rev}
+            - cargo test --verbose --all-features
+          mounts:
+            - content:
+                url: https://win.rustup.rs/
+              directory: rust-init
+              format: exe
```

Comment:
I think in this case you'd need a `file` mount (see docs at https://docs.taskcluster.net/docs/reference/workers/generic-worker/multiuser-windows-payload).
Something like:
```
- content:
    url: https://win.rustup.rs/
  file: rust-init.exe
```

Reply:
Oh, thank you! :)

If I want to put the exe inside a directory, do I need to create it or I can just set this one below?

```
- content:
    url: https://win.rustup.rs/
  file: rust-init\rust-init.exe
```